protocol/contracts/v2/strategies/RamosStrategy.sol

```diff
@@ -51,12 +54,22 @@ contract RamosStrategy  is AbstractStrategy, IRamosProtocolTokenVault {
         templeToken.mint(recipient, amount);
     }
 
+    function borrowQuoteToken(uint256 amount, address recipient) external {
+        treasuryReservesVault.borrow(amount, address(this));
```

Comment:
You can just do this in one step, since there's a recipient address in `trv.borrow()`

`treasuryReservesVault.borrow(amount, recipient)`

Doing it as two steps costs more in gas